mbuild/conversion.py

```diff
@@ -59,22 +60,30 @@ def load(filename_or_object,
     rigid : bool, optional, default=False
         Treat the compound as a rigid body
     backend : str, optional, default=None
-        Backend used to load structure from file. If not specified, a default
-        backend (extension specific) will be used.
+        Backend used to load structure from file or string. If not specified,
+        a default backend (extension specific) will be used.
     smiles: bool, optional, default=False
```

Comment:
might be nice if we let users know which backend will be used by default

Reply:
agree

Reply:
added below